docs/tempo/website/getting-started/_index.md

```diff
@@ -3,11 +3,12 @@ title: Getting started
 weight: 100
 ---
 
-Getting started with Tempo is easy.  For an application already instrumented for tracing, this guide can help quickly set it up with Tempo. If you're looking for a demo application to play around with Tempo, skip to the [examples](#examples-with-demo-app) section at the bottom of this page.
+# Getting started with Tempo
 
-## Setting up Tempo
+Getting started with Tempo is easy.  For an application already instrumented for tracing, this guide can help quickly set it up with Tempo. If you're looking for a demo application to play around with Tempo, skip to the [examples with demo app]({{< relref "example-demo-app.md" >}})topic.
```

Comment:
nit: We need a space between the link and the following word here - `[examples with demo app]({{< relref "example-demo-app.md" >}})topic`

Reply:
@joe-elliott, did you fix the nits @annanay25 pointed out? LMK, thanks!

Reply:
@achatterjee-grafana i did not.  merged blindly!

feel free to submit a new PR :)